haskell-style.md

```diff
@@ -234,35 +234,23 @@ Sort your exports, unless the order matters in your desired Haddock output.
 
 **Automated**: sorting and formatting of extension pragmas.
 
-- All Haskell packages MUST use the following `default-extensions`:
+- All Haskell packages MUST use the following language version and `default-extensions`:
```

Comment:
Thoughts on:

```suggestion
- All Haskell packages MUST use the following `language` and `default-extensions`:
```

?

If we're going to speak of one of them as `code`, might as well do them both that way.